src/TinyIoC/TinyIoC.cs

```diff
@@ -4246,7 +4246,7 @@ public static Assembly Assembly(this Type type)
     }
 #endif
     // reverse shim for WinRT SR changes...
-#if (!NETFX_CORE && !PORTABLE && !NETSTANDARD1_0 && !NETSTANDARD1_1 && !NETSTANDARD1_2 && !NETSTANDARD1_3 && !NETSTANDARD1_4 && !NETSTANDARD1_5 || !NETSTANDARD1_6)

```

Comment:
This and the `ReverseTypeExtender` above were both being compiled in and colliding